src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -214,6 +214,22 @@ internal void SetExperimentalFeatures(ConfigScope scope, string featureName, boo
             }
         }
 
+        internal bool IsImplicitWinCompatEnabled()
+        {
+            bool? settingValue = ReadValueFromFile<bool?>(ConfigScope.CurrentUser, "ImplicitWinCompatEnabled", null);
```

Comment:
If this is an opt-out behavior, then the setting key should be `DisableImplicitWinCompat`, and default value is false.

Reply:
@joeyaiello Can you please review the setting key name?

Reply:
Changed setting name to `DisableImplicitWinCompat` with False as default.